docs/api-types/issuer/spec.md

```diff
@@ -126,12 +126,40 @@ clouddns:
 ```yaml
 route53:
   accessKeyID: AKIAIOSFODNN7EXAMPLE
+  hostedZoneID: DIKER8JPL21PSA
```

Comment:
Can we remove this from the example? This field is likely going to be removed (and moved to the Certificate resource), and is not the recommended way to create an issuer configured for solving DNS with Amazon (as it ties to Issuer to a single dns zone)